cli/src/prepareProject.ts

```diff
@@ -240,20 +240,29 @@ function copyChangelog(
 function copyMoonwaveFolder(
   projectDir: string,
   tempDir: string
-): { customCssExists: boolean } {
+): { customCssExists: boolean, customSidebarsExists: boolean } {
   const staticDir = path.join(projectDir, ".moonwave", "static")
   if (fs.existsSync(staticDir)) {
     fs.copySync(staticDir, path.join(tempDir, "static"))
   }
 
+  let customCssExists = false, customSidebarsExists = false;
+
```

Comment:
Can we replace this with something like...

```js
const status = { customCssExists: false, customSidebarsExists: false }
````

Then, change `status.customCssExsists = true` below.

Then just return `status`